src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+import java.text.DecimalFormat;
+
+public abstract class Figure implements Quadrangular, Triangular {
```

Comment:
```suggestion
public abstract class Figure implements AreaCalculator, Drawable {
```

create required interfaces and methods